construo/src/main/java/io/github/fourlastor/construo/ConstruoPluginExtension.kt

```diff
@@ -52,6 +54,7 @@ interface JlinkOptions {
     val modules: ListProperty<String>
     val guessModulesFromJar: Property<Boolean>
     val includeDefaultCryptoModules: Property<Boolean>
+    val multiReleaseVersion: Property<String?>
```

Comment:
No need to make this nullable and set a default `null` value.
Simply not setting a convention will make it empty by default, which is an acceptable value in `CreateRuntimeImageTask` (as it's marked as `@Optional`).

Remember to also change the property of `CreateRuntimeImageTask` to be `Property<String>` when you change this